azext_iot/_params.py

```diff
@@ -819,6 +819,13 @@ def load_arguments(self, _):
             type=int,
             help="Maximum number of configurations to return. By default all configurations are returned.",
         )
+        context.argument(
+            "custom_metric_queries",
+            options_list=["--custom-metric-queries"],
+            type=dict,
+            help="Map of custrom metric queries for configuration. "
```

Comment:
custom

Reply:
also this can be a file too - use wording like this example for az dt model create 
![image](https://user-images.githubusercontent.com/73560279/209995966-b81225d5-81b2-4b78-90fc-f32c742f9a5c.png)

Reply:
true! forgot about the file, thanks!